### DISCUSSION TITLE
Weird Bug I've encountered

### DISCUSSION THREAD

**Discussion by RunninPigeon** [TrianglyRU/OrbinautFramework#30] [Q&A] [answered]
I just loaded into the zone and...
![Orbinaut Framework 2 9_7_2024 9_24_39 PM](https://github.com/user-attachments/assets/61cc16bf-bb16-4299-a6b3-631d36830bd8)
There should be some rings and a monitor here.
![Orbinaut Framework 2 9_7_2024 9_24_55 PM](https://github.com/user-attachments/assets/48dccc80-fca5-481f-82d3-cb6589a3d926)
Unloading these objects by walking off-screen, and walking back in makes them appear again.
Is there anyway to fix this with _all_ objects?

**Comment by TrianglyRU** [comment]
Cannot replicate. Look at TSZ and how objects are placed there. They all should be on the same one layer with the player, different from Controller layer

**Reply by RunninPigeon** [reply]
They're on the same layer, i'll go ahead and give you the project file so you can see what the issue is: https://www.mediafire.com/file/5f2kux7rrntxu0m/OrbinautFramework2.yyz/file

**Reply by TrianglyRU** [reply, marked as answer]
Confirmed as a bug, will be fixed with the release of 2024.10. Avoid placing objects off-camera near the start of the stage for now

**Reply by RunninPigeon** [reply]
Found another bug when using the 2024.10 version, dying and using debug mode causes objects to not load in
![Orbinaut Framework 2 9_8_2024 6_35_19 PM](https://github.com/user-attachments/assets/e865c5a9-dd4c-4eee-883a-261e1a2f971b)

**Reply by RunninPigeon** [reply]
Pausing a lot also causes this to happen too?
